app/containers/message/Attachments.tsx

```diff
@@ -51,6 +52,17 @@ const Attachments = React.memo(
 			if (file.actions && file.actions.length > 0) {
 				return <AttachedActions attachment={file} theme={theme} />;
 			}
+			if (file.collapsed)
+				return (
+					<CollapsibleQuote
+						key={index}
+						index={index}
+						attachment={file}
+						timeFormat={timeFormat}
+						getCustomEmoji={getCustomEmoji}
+						theme={theme}
```

Comment:
As `CollapsibleQuote` is a new component, I wonder if we should use `useTheme` instead of prop drilling (as we're doing on the other components)